synapse/lib/storm.py

```diff
@@ -1085,13 +1069,13 @@ def _stormOperPivot(self, query, oper):
 
         [query.add(t)for t in self.stormTufosBy('in', dstp, list(vals), limit=limit)]
 
-    def _stormOperNextSeq(self, query, oper):
+    def _stormOperNextTag(self, query, oper):
         name = None
 
         args = oper[1].get('args')
         kwargs = dict(oper[1].get('kwlist'))
 
-        doc = kwargs.get('doc')
+        doc = kwargs.get('doc', '??')
```

Comment:
I hardcoded in the default value for tag docs here as passing in None raised exception.